pandas/tests/io/pytables/common.py

```diff
@@ -0,0 +1,74 @@
+from contextlib import contextmanager
+import os
+import tempfile
+
+from pandas.io.pytables import HDFStore
+
+
+def safe_remove(path):
+    if path is not None:
+        try:
+            os.remove(path)
+        except OSError:
+            pass
+
+
+def safe_close(store):
```

Comment:
in a future PR, we want to have doc-strings & typing for these

Reply:
type errors in tests directories are not reported by mypy

https://github.com/pandas-dev/pandas/blob/75c9783d4924c98d84e9722060686fc7b4643259/setup.cfg#L133-L134

so should typing be added? does help grok functions but equally if not correct (since not checked) could be confusing.